tests/functional/ctlplane/openstackoperator_controller_test.go

```diff
@@ -576,6 +576,7 @@ var _ = Describe("OpenStackOperator controller", func() {
 			Expect(OSCtlplane.Spec.Manila.APIOverride.Route).Should(Not(BeNil()))
 			Expect(OSCtlplane.Spec.Manila.APIOverride.Route.Annotations).Should(HaveKeyWithValue("haproxy.router.openshift.io/timeout", "60s"))
 			Expect(OSCtlplane.Spec.Manila.APIOverride.Route.Annotations).Should(HaveKeyWithValue("api.manila.openstack.org/timeout", "60s"))
+			Expect(OSCtlplane.Spec.Octavia.APIOverride.Route.Annotations).Should(HaveKeyWithValue("haproxy.router.openshift.io/timeout", "120s"))
```

Comment:
This will cause a `nil pointer dereference` in the test suite since Octavia is not enabled in functional tests.

I don't think we should make this PR about enabling Octavia in functional tests, since that could be quite complex. So I would suggest that we remove this line for now, and open a new Story for enabling Octavia in the functional tests.

Reply:
Yeah make totally sense. Let me remove this line and recover it when Octavia will be enabled for FTs.